index.html

```diff
@@ -1,40 +1,62 @@
-<html>
-    <head>
-        <title>PyData Chicago</title>
-        <link href="styles/style.css" rel="stylesheet" type="text/css"/>
-        <link href="styles/bootstrap.min.css" rel="stylesheet" type="text/css"/>
-    </head>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>PyData Chicago</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="styles/bootstrap.min.css" rel="stylesheet">
+    <link href="styles/style.css" rel="stylesheet" type="text/css"/>
+
+    <!-- Custom styles for this template -->
+    <!--<link href="styles/jumbotron-narrow.css" rel="stylesheet">-->
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+</head>
+
+<body>
+
+<div class="container">
+    <div class="header clearfix">
 
-    <body>
         <div class="container text-center">
             <a href="/" class="header-link">
                 <strong>Meetup</strong>
             </a>
             <a href="http://pydata.org/chicago2016/" class="header-link">
```

Comment:
We can probably remove this link altogether.

Reply:
Would work fine if we remove the tag altogether